apps/site/assets/css/_footer.scss

```diff
@@ -340,12 +360,13 @@ address {
 .lower-footer {
   @include media-breakpoint-down(xs) {
     flex-direction: column;
+    margin-left: 16px;
+    margin-right: 16px;
```

Comment:
Can you either move the HTML element with the copyright info into this container, or add this margin CSS to the copyright? 
<img width="424" alt="image" src="https://user-images.githubusercontent.com/2136286/189691705-64d857b4-6be9-43ec-b64e-0d91a42c1bdf.png">

Reply:
Also looks like we need some space to the right in the lower footer for the tablet sizes... this screenshot is at 768px wide. 
<img width="835" alt="image" src="https://user-images.githubusercontent.com/2136286/189692612-9e2bc810-0aba-4bfa-9963-9b2e03ccbbd0.png">